CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## Version 1.1.0
+
+### Changed
+Create, Read, Rename and Delete of attachments are not supported if repository is versioned.
```

Comment:
It should be added not changed.
The message could be that "Added additional validation to check if repository is versioned and show UI message that versioned repository is not supported".

Reply:
Done